docs/user-guide/scanning-registries.md

```diff
@@ -152,7 +153,61 @@ spec:
       variant: "v7"
 ```
 
-## 5. Monitor Scan Progress
+## 5. Filtering By Tags
+
+Another way to reduce the scanning load is to apply filters to the image tags. To use this filter, you have to use the `matchConditions` field, provided under `repositories`.
+
+This field allows the user to write their own [CEL](https://kubernetes.io/docs/reference/using-api/cel/) expressions to filter for image tags. Let's see an example:
```

Comment:
```suggestion
You can filter image tags using CEL](https://kubernetes.io/docs/reference/using-api/cel/) expressions in the `matchConditions` field under `repositories`. This lets you use regex patterns, semantic versioning comparisons, or string operations (like substring matching or contains checks) to select which tags to scan. This filtering capability helps reduce the number of scanned images and lowers the load on the system.

Let's see an example:
```

I rephrased the paragraph to focus more on the feature rather than the load of the system